app/uk/gov/hmrc/economiccrimelevyregistration/models/AmlSupervisorType.scala

```diff
@@ -52,9 +52,7 @@ object AmlSupervisorType {
     messages: Messages
   ): Seq[RadioItem] = {
     val amlProfessionalBodySupervisorOptions: Seq[SelectItem] =
-      SelectItem(
-        text = ""
-      ) +: appConfig.amlProfessionalBodySupervisors.map { opb =>
```

Comment:
This is necessary for the accessible autocomplete to behave the way it should - it's a limitation of the component, is there a reason you removed it?

Reply:
Edit: I know why you have done this - to solve the issue: 'Element “option” without attribute “label” must not be empty.'

In this case we need to leave this one alone - as the accessible autocomplete component requires it